package.json

```diff
@@ -52,18 +50,23 @@
       }
     ],
     "menus": {
+      "explorer/context": [
+        {
+          "command": "scope-to-this.scope",
+          "group": "navigation"
+        }
+      ],
       "view/title": [
         {
           "command": "scope-to-this.clear",
-          "group": "navigation",
-          "when": "view == 'workbench.explorer.fileView' && scopeToThis.scoped"
```

Comment:
I highly encourage to reduce unnecessary diffs. Since this is just a reorder, please revert